tests/apps/courses/test_templatetags_extra_tags_course_enrollment_widget_props.py

```diff
@@ -20,19 +24,25 @@ def test_course_enrollment_widget_props_tag(self):
         course_enrollment_widget_props should return these properties wrapped into
         a courseRun object as a stringified json.
         """
-        course_run = CourseRunFactory(
-            resource_link="http://example.edx:8073/courses/course-v1:edX+DemoX+Demo_Course/course/"
-        )
-        context = {"run": course_run}
-        self.assertEqual(
-            course_enrollment_widget_props(context),
-            json.dumps(
-                {
-                    "courseRun": {
-                        "id": course_run.id,
-                        "resource_link": course_run.resource_link,
-                        "priority": course_run.direct_course.state["priority"],
+        now = timezone.now()
+        with mock.patch.object(timezone, "now", return_value=now):
```

Comment:
I would say that for clarity of intention, and avoiding side effects, this mock should only wrap the call to `course_enrollment_widget_props`

Reply:
Is it ok like this, or should I extract `course_enrollment_widget_props(context),` from assertEqual to wrap it alone in `course_enrollment_widget_props` ?